packages/governance/src/votes/votes.cairo

```diff
@@ -180,21 +259,54 @@ pub mod ERC20VotesComponent {
     // Internal
     //
 
+    impl ERC721VotesImpl<
+        TContractState,
+        +HasComponent<TContractState>,
+        +SRC5Component::HasComponent<TContractState>,
+        impl ERC721: ERC721Component::HasComponent<TContractState>,
+        +ERC721Component::ERC721HooksTrait<TContractState>,
+        +Drop<TContractState>
+    > of VotingUnitsTrait<ComponentState<TContractState>> {
+        /// Returns the number of voting units for a given account.
+        ///
+        /// This implementation is specific to ERC721 tokens, where each token
+        /// represents one voting unit. The function returns the balance of
+        /// ERC721 tokens for the specified account.
+        fn get_voting_units(
+            self: @ComponentState<TContractState>, account: ContractAddress
+        ) -> u256 {
+            let erc721_component = get_dep_component!(self, ERC721);
+            erc721_component.balance_of(account).into()
+        }
+    }
+
+    impl ERC20VotesImpl<
```

Comment:
Should we put ERC20VotesImpl before ERC721VotesImpl for better organization?

Reply:
better alphabetical organization you mean? I don't have a strong opinion here.

Reply:
It feels more consistent since it is presented in that order everywhere where both tokens are mentioned/used together (docs, comments, etc...).

Reply:
I don't think it has a big effect, but doesn't hurt to change the order